pkg/service/utgen/assets/test_generation.toml

```diff
@@ -128,14 +212,15 @@ new_tests:
 {%- endif %}
   test_code: |
 {%- if language in ["python"] %}
+    import pytest
+    @pytest.mark.asyncio
     def ...
```

Comment:
why are we making this change?

Reply:
Pytest wasnt. being added, i guess the comment below will solve the problem of async def functions